.github/workflows/package.yml

```diff
@@ -46,6 +50,39 @@ jobs:
         id: build-debian-unstable
         with:
           args: --no-sign
+          ppa: ${{ steps.ppa.outputs.ppa }}
```

Comment:
Debian builds should not be referencing Ubuntu PPAs. The packages are published independently at https://packages.debian.org/search?keywords=libzim.
Requesting @legoktm to upload libzim9 to the other Debian versions (only available on experimental so far).

Reply:
> Debian builds should not be referencing Ubuntu PPAs.

Fixed